source/server/getting-started/bungeecord.rst

```diff
@@ -1,63 +1,72 @@
-Using Sponge with BungeeCord
```

Comment:
Maybe also change the filename to server proxies?

Reply:
I thought about it, but it might break links that people have posted elsewhere. I don't mind doing it, but 'tis up to you (and/or others).

Reply:
do you have the option to redirect old links to the new page? otherwise i would say leave the old link intact

Reply:
Chances are most users will search for "bungee" and not "server proxy" anyway.